### ISSUE TITLE
The merge filters option stopped working

### ISSUE THREAD

**Issue by Dario1404** [RealRaven2000/quickFilters#262]
Hi after upgrading to betterbird (the problem occurs even with the latest Thunderbird release) the filter merge option stopped working here is the debug log

```log
quickFilters  quickFilters.Init()  
        Assistant active: true
        BrowserInfo: [object Object]
      qFilters-utils.js:786:12
quickFilters [logTime init]
 selectTemplateFromListTimer() qFilters-utils.js:744:13
quickFilters {FILTERLIST} 11:57:11.915  [6790 ms]   
 patchToolbarTheme(activate) 
Object { win: ChromeWindow chrome://messenger/content/FilterListDialog.xhtml, doc: HTMLDocument chrome://messenger/content/FilterListDialog.xhtml, toolbarId: "quickfilters-toolbar" }
 
activate { target: ChromeWindow chrome://messenger/content/FilterListDialog.xhtml, isTrusted: true, srcElement: ChromeWindow chrome://messenger/content/FilterListDialog.xhtml, eventPhase: 0, bubbles: true, cancelable: true, returnValue: true, defaultPrevented: false, composed: false, timeStamp: 27431, … }
qFilters-utils.js:816:13
Uncaught (in promise) ReferenceError: assignment to undeclared variable isMerge
    merge chrome://quickfilters/content/qFilters-list.js:377
    oncommand chrome://messenger/content/FilterListDialog.xhtml:1
qFilters-list.js:377:7
quickFilters {FILTERLIST} 11:57:12.419  [504 ms]   
 patchToolbarTheme(activate) 
Object { win: ChromeWindow chrome://messenger/content/FilterListDialog.xhtml, doc: HTMLDocument chrome://messenger/content/FilterListDialog.xhtml, toolbarId: "quickfilters-toolbar" }
 
activate { target: ChromeWindow chrome://messenger/content/FilterListDialog.xhtml, isTrusted: true, srcElement: ChromeWindow chrome://messenger/content/FilterListDialog.xhtml, eventPhase: 0, bubbles: true, cancelable: true, returnValue: true, defaultPrevented: false, composed: false, timeStamp: 30469.765, … }
qFilters-utils.js:816:13
Questa pagina si trova in modalità Quirks. Questo potrebbe avere effetto sul layout della pagina. Per la modalità standard utilizzare “<!DOCTYPE html>”. 4 qFilters-list.js

```

**Comment by RealRaven2000** [comment]
Please try this quick fix:

[quickFilters-wx-6.5.3pre4.zip](https://github.com/user-attachments/files/16760683/quickFilters-wx-6.5.3pre4.zip)

----

To try the version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)

**Comment by Dario1404** [comment]
> Please try this quick fix:
> 
> [quickFilters-wx-6.5.3pre4.zip](https://github.com/user-attachments/files/16760683/quickFilters-wx-6.5.3pre4.zip)
> 
> To try the version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)

Thanks for your timeliness the fix solved

**Comment by Dario1404** [comment]
solved

**Comment by RealRaven2000** [comment]
Since this bug affects everyone, I leave it open until the new version is released to my users.

**Comment by RealRaven2000** [comment]
I made a mistake with the version number - here is the latest version. (Otherwise you won't get automatic updates with the official version). this one is what is claims (6.5.3 prerelease 6 ). Also added change log and a sale / bargain, because it's been a while.

[quickFilters-wx-6.5.3pre6.zip](https://github.com/user-attachments/files/16762066/quickFilters-wx-6.5.3pre6.zip)

To see the (fully localized) change log please click the appropriate item on the quickFilters menu:

![image](https://github.com/user-attachments/assets/95893fcc-e4b9-4b29-87c2-bad718b9f1a1)

---

To install the version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)

**Comment by RealRaven2000** [comment]
Implemented in 6.5.3 published on 27/08/2024
